### PR TITLE
Fixes TimeUI update on seek

### PR DESCRIPTION
**Pull request by shahlabs**
## Description
TimeUpdate ui does not fire on seek when paused. 

## Specific Changes proposed
By manually triggering a timeupdate we force the ui to not use the cache.

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/control-bar/progress-control/seek-bar.js [hunk 1]
```diff
@@ -244,6 +244,8 @@ class SeekBar extends Slider {
     super.handleMouseUp(event);
 
     this.player_.scrubbing(false);
+    // forces timeui to not use cache
+    this.player_.tech_.trigger('timeupdate');
```
Comment (gkatsev) [inline src/js/control-bar/progress-control/seek-bar.js]: is it necessary to trigger on the tech or can we trigger it on the player itself?

Also, we should trigger this event like we do here: https://github.com/videojs/video.js/blob/master/src/js/tech/tech.js#L304-L310
and we should copy over and update the jsdoc as well like there.

Comment (shahlabs) [reply src/js/control-bar/progress-control/seek-bar.js]: It is necessary to trigger on the tech, I had tried `this.player_.trigger`, it didn't work. I will update the trigger event.

Comment (gkatsev) [reply src/js/control-bar/progress-control/seek-bar.js]: cool, thanks @shahlabs!

Comment (shahlabs) [reply src/js/control-bar/progress-control/seek-bar.js]: Retested with this.player_.trigger and it worked, so updating the trigger on player @gkatsev

**Comment by gkatsev** [comment]
Also, I updated the jsdoc comment.

**Pull request merged**